### PR TITLE
chore: format

### PR DESCRIPTION
**Pull request by vinckr**

### CODE REVIEW

File: templates/repository/common/CONTRIBUTING.md [hunk 1]
```diff
@@ -249,7 +250,7 @@ community a safe place for you and we've got your back.
 - Private harassment is also unacceptable. No matter who you are, if you feel
   you have been or are being harassed or made uncomfortable by a community
   member, please contact one of the channel ops or a member of the ORY
-  ${PROJECT} core team immediately.
+  \${PROJECT} core team immediately.
```
Comment (aeneasr) [inline templates/repository/common/CONTRIBUTING.md:253]: These `\` do not belong here. I think it should work with `$PROJECT` also but please check yourself.

**Review by aeneasr — CHANGES_REQUESTED** [review]

File: templates/repository/common/CONTRIBUTING.md (hunk 1 above)
Comment (vinckr) [reply templates/repository/common/CONTRIBUTING.md:253]: I know they dont belong but 
`npm run format `
put them there. 
If I remove them the format check fails.

Comment (vinckr) [reply templates/repository/common/CONTRIBUTING.md:253]: I will check if it works without {  } but I am having some trouble getting sync.sh to substitute anything at the moment. 
There was no change to the script and it worked fine before, but now nothing gets substituted...
Have to look more into it I guess.

Comment (vinckr) [reply templates/repository/common/CONTRIBUTING.md:253]: This is my console output, testing just the substitution. 
```
+ branch=master
+ humanName=meta
+ project=meta
+ REPOSITORY=meta
+ discussions=https://github.com/meta/discussions
++ find CONTRIBUTING.md -type f -print
+ for f in '$(find "CONTRIBUTING.md" -type f -print)'
+ env -i DISCUSSIONS=https://github.com/meta/discussions PROJECT=meta REPOSITORY=meta /bin/bash -c 'envsubst < "CONTRIBUTING.md" | sponge "CONTRIBUTING.md"'
```

I changed the env variables order and only substitute in CONTRIBUTING.md to test, but should work the same.

For example I go to ory/web and run `sh test.sh master web web web`, nothing happens :/. 
Before I did it exactly the same way and it worked. 
```
#!/bin/bash

set -Eexuo pipefail

branch=$1
humanName=$2
project=$3
REPOSITORY=$4
discussions="https://github.com/${REPOSITORY}/discussions"

    for f in $(find "CONTRIBUTING.md" -type f -print); do
        env -i DISCUSSIONS="https://github.com/meta/discussions" PROJECT="$humanName" REPOSITORY="$project" /bin/bash -c "envsubst < \"$f\" | sponge \"$f\""
    done
```

If you see a glaring error let me know, I will look further what went wrong...

Comment (aeneasr) [reply templates/repository/common/CONTRIBUTING.md:253]:
```
$PROJECT
```

should work. Execute ` env -i DISCUSSIONS="https://github.com/meta/discussions" PROJECT="$humanName" REPOSITORY="$project" /bin/bash -c "envsubst < \"$f\" | sponge \"$f\""` on a file directly by e.g.:

```
f="path/to/file"
env -i DISCUSSIONS="https://github.com/meta/discussions" PROJECT=abc REPOSITORY=def /bin/bash -c "envsubst < \"$f\" | sponge \"$f\""
```

**Comment by aeneasr** [comment]
What is the status of this?

**Comment by vinckr** [comment]
Just getting the PR together!

**Pull request closed without merging**